### PR TITLE
Add Reverse String

### PR DESCRIPTION
**Pull request by iHiD**
I wasn't sure on the best module name. Do we have a standard we use for Practice Exercises?

### CODE REVIEW

File: exercises/practice/reverse-string/.meta/solutions/reverse_string.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module ReverseString
```
Comment (kotp) [inline exercises/practice/reverse-string/.meta/solutions/reverse_string.rb:1]: We might name it similar to other modules in Ruby, with "able" at the end.  But we do not have to.

**Review by kotp — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
rubocop is complaining

File: exercises/practice/reverse-string/reverse_string.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+=begin
+Write your code for the 'Reverse String' exercise in this file. Make the tests in
+`reverse_string_test.rb` pass.
+
+To get started with TDD, see the `README.md` file in your
+`ruby/reverse-string` directory.
+=end
+
```
Comment (kotp) [inline exercises/practice/reverse-string/reverse_string.rb:8]:
```suggestion
=end
```

**Comment by kotp** [comment]
There are only two practice exercises that look like this:

```
exercises/practice/reverse-string/.meta
├── config.json                        
├── solutions                          
│   └── reverse_string.rb              
└── tests.toml                         
```

The rest of them look like:

```
exercises/practice/accumulate/.meta
├── config.json                    
├── example.rb                     
└── tests.toml                     
```

**Comment by iHiD** [comment]
@kotp Thanks. Fixed here and updated for the long-term fix here: https://github.com/exercism/ruby/pull/1628

Please merge if you're happy with this :)

File: exercises/practice/reverse-string/.meta/config.json [hunk 3]
```diff
@@ -4,13 +4,13 @@
   ],
   "files": {
     "solution": [
-      "reverse_string.rb"
+      "reverser.rb"
```
Comment (kotp) [inline exercises/practice/reverse-string/.meta/config.json:7]: Generally speaking, though it is not always the case, the file name has matched the slug of the exercise  (with the exception of the `-` being `_` when representing two words.

File: exercises/practice/reverse-string/.meta/example.rb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-module ReverseString
+module Reverser
```
Comment (kotp) [inline exercises/practice/reverse-string/.meta/example.rb:1]: I like it.

**Review by kotp — APPROVED** [review]

**Comment by kotp** [comment]
> @kotp Thanks. Fixed here and updated for the long-term fix here: #1628
> 
> Please merge if you're happy with this :)

The file name change that is a bit more than a approve/like/merge.  It speaks to "convention" from the Exercism viewpoint.  So, I would like to confirm with you that the difference (though it does happen) in the naming style is also fine with @ErikSchierboom as well as you.

**Review by ErikSchierboom — APPROVED** [review]
Lgtm!

**Pull request merged**